src/lib/OTA/OTA.h

```diff
@@ -64,7 +64,9 @@ typedef struct {
         } PACKED dbg_linkstats;
         /** PACKET_TYPE_MSP **/
         struct {
-            uint8_t packageIndex;
+            uint8_t packageIndex:5,
+                    tlmFlag:1,
+                    spare:2;
```

Comment:
I think this breaks compatibility with V3 using this packet.  If the spare and tlmflag are moved up and the MSB in the bytes then we might be safe.

Something to test...